ksql-common/src/main/java/io/confluent/ksql/function/AggregateFunctionFactory.java

```diff
@@ -32,7 +32,7 @@
 
   public AggregateFunctionFactory(final String functionName,
                                   final List<KsqlAggregateFunction<?, ?>> aggregateFunctionList) {
-    this(new UdfMetadata(functionName, "", "confluent", "", KsqlFunction.INTERNAL_PATH),
+    this(new UdfMetadata(functionName, "", "confluent", "", KsqlFunction.INTERNAL_PATH, false),
```

Comment:
Is `UdfMetadata` used for both UDF and UDAF?
Should we consider renaming it to reflect this?

Reply:
I think `UdfMetadata` is probably ok. They are all User defined Functions of some sort